src/base/bittorrent/session.cpp

```diff
@@ -1837,13 +1837,16 @@ bool Session::deleteTorrent(const QString &hash, bool deleteLocalFiles)
         QString rootPath = torrent->rootPath(true);
         if (!rootPath.isEmpty())
             // torrent with root folder
-            m_savePathsToRemove[torrent->hash()] = rootPath;
+            m_removedTorrents[torrent->hash()] = {torrent->name(), rootPath, true};
```

Comment:
Shouldn't you use  `deleteLocalFiles` here instead of `true`?

Reply:
Sure I can do that.